README.md

```diff
@@ -8,35 +8,23 @@ The LDES Server is a configurable component that can be used to ingest, store, t
 We'll show you how to set up your own LDES Server both locally and via Docker using a mongo db for storage and a rest endpoint for ingestion and publication.
 Afterwards, you can change storage, ingestion and publication options by plugging in other components.
 
-### Locally
+## Locally
 
-#### Docker-compose
+### Docker-compose
 
 There are 3 files where you can configure the dockerized application:
-- [The .env file](#docker-compose-config-env)
 - [The config files](#docker-compose-config-config)
 - [The docker-compose file](#docker-compose-config-yml)
 
-##### <a name="docker-compose-config-env"></a>The .env file
-
-The [.env](.env) file allows you to configure the repository for the LDES bom (e.g. if you've forked the VSDS repo and want to test changes) and change the port the application will run on (in case you already have services running on the default port 8080).
-
-```bash
-# The LDES BOM repository
-DOCKER_LDES_BOM_REPO=https://github.com/Informatievlaanderen/VSDS-LDES.git
-# Run the LDES server app on this port
-DOCKER_LDES_SERVER_PORT=6060
-```
-
-##### <a name="docker-compose-config-config"></a>The config files: [config.env](docker-compose/config.env) or [config.local.env](docker-compose/config.local.env)
+#### <a name="docker-compose-config-config"></a>The config files: [config.env](docker-compose/config.env) or [config.local.env](docker-compose/config.local.env)
 
 Runtime settings can be defined in the configuration files. Use [config.env](docker-compose/config.env) for a public setup and be sure not to commit this file, as it contains secrets. For a local setup, use [config.local.env](docker-compose/config.local.env). It's safe to commit this file with secrets, since they'll only be relevant to your local environment (assuming you don't reuse personal passwords for public services for your local services).
```

Comment:
It is never safe to commit files with secrets. But probably I am mis understanding the concept of local commit. Local creation without this file being part of git I might understand.

Reply:
Should it then be a good practice to add the config.local.env and config.env to the .gitignore file?

Reply:
Maybe explain a bit more about the configuration options in these config files. Some are self explainable others or not. 
So what happens if no LDES configuration is set in the config.local.env?  Apparently also defaults are used but it is not clear which they are from within the readme, config files